newIDE/app/src/Export/LocalExporters/LocalPreviewLauncher/index.js

```diff
@@ -38,6 +38,7 @@ type State = {|
     title: string,
     backgroundColor: string,
   },
+  hideMenubar: boolean,
```

Comment:
Nitpicking: Everywhere in the code, this should be `MenuBar` (with a capital B). Menubar is not a word, it's two different words.